Cohort3/menu hover

Edu-Fortes opened the pull request:
Nessa tarefa modifiquei os ícones que estavam como IMG por svg markup. Assim consegui alterar as cores dos vetores sem trocar por outra imagem. Efeito de hover aplicado utilizando "scale" transition. Toda interação do hover acontece na classe "menu__item", acredito que assim fica mais fácil para futuras manutenções.
![hover_320px](https://user-images.githubusercontent.com/46508709/214304755-9bfec850-bec0-4191-ba93-b813d1425f14.jpg)
![hover_1750px](https://user-images.githubusercontent.com/46508709/214304760-008f1cf9-cf12-49d5-b46d-d91d9a42c74e.jpg)

Edu-Fortes committed:
Replace IMG tag for icons to SVG Markup

Edu-Fortes committed:
Add hover function on menu itens

Edu-Fortes committed:
Remove style.css and style.css link ref in HTML

Edu-Fortes committed:
Add modifications sugested by Puntar um menu-categorias

Edu-Fortes committed:
feat(hover): change word color when hovering buttons

thiagopuntar reviewed (changes requested):
Tá tudo perfeito Eduardo, parabéns pelo excelente trabalho.
A única coisa que tá faltando é que queremos que ao fazer hover no button tanto o svg quanto o span fique com a cor do hover. No momento, o span só fica com hover quando ele é diretamente focado.

Edu-Fortes committed:
hotfix(menu-hover): corret hover in button

Alter class name on button to reflect corret description removing menu__item class. Remove color from menu__paragraph class to make hover work the right way. Puntar correction suggestion.

Edu-Fortes commented:
> Tá tudo perfeito Eduardo, parabéns pelo excelente trabalho. A única coisa que tá faltando é que queremos que ao fazer hover no button tanto o svg quanto o span fique com a cor do hover. No momento, o span só fica com hover quando ele é diretamente focado.

Fiz a alteração sugerida e realmente está se comportando do jeito correto. Alterei também o nome da classe do botão para ficar mais fácil de identificar.

Edu-Fortes committed:
refact(menu-hover): change button class

Change button class from button to btn-menu. Making button class available to other section
